src/api/java/baritone/api/utils/ExampleBaritoneControl.java

```diff
@@ -368,6 +369,82 @@ public boolean isInGoal(int x, int y, int z) {
             logDirect("tunneling");
             return true;
         }
+        if(msg.startsWith("tunnelcustom"))
+        {
+            String suffix = msg.substring("tunnelcustom".length());
+            int a;
+            int b;
+            int c;
+            if (suffix.isEmpty()) {
+                    logDirect("Need to specify depth, height and width of the tunnel. eg .b tunnel2 depth height width");
+                    return true;
+            }
+            else {
+                try {
+                    String[] spl = suffix.split(" ");
+                    if(spl.length != 4) {
+                        logDirect(Integer.toString(spl.length));
+                        logDirect("Please enter 3 values");
+                        return true;
+                    }
+                    a = Integer.parseInt(spl[1]);
+                    b = Integer.parseInt(spl[2]);
+                    c = Integer.parseInt(spl[3]);
+                } catch (NumberFormatException | ArrayIndexOutOfBoundsException | NullPointerException ex) {
+                    logDirect("unable to parse");
```

Comment:
Should probably Make this error message `One or more number arguments were invalid` or something along the lines of this.